cosmo_tester/test_suites/image_based_tests/cfy_manager_test.py

```diff
@@ -33,6 +40,21 @@
 ''' % MQ_PASSWORDS_PATH
 
 
+@pytest.fixture(scope='function')
+def manager_5_1_0(request, ssh_key, module_tmpdir, test_config, logger):
+    hosts = Hosts(ssh_key, module_tmpdir, test_config, logger, request,
+                  bootstrappable=True)
+    hosts.instances[0].image_name = test_config['upgrade']['manager_image']
+
+    hosts.create()
+    hosts.instances[0].wait_for_ssh()
+    hosts.instances[0].bootstrap()
```

Comment:
Also, if you use the actual 5.1.0 image then you don't need to bootstrap at all.

Reply:
Replied to it in my first reply.